### DISCUSSION TITLE
ReadBladeFile:Invalid numerical input for file ".\5MW_Baseline/NRELOffshrBsline5MW_Blade_Fault2.dat" occurred while trying to read BldFl1Sh(1)

### DISCUSSION THREAD

**Discussion by Crystal0912** [OpenFAST/openfast#1478] [Q&A]

**Comment by jjonkman** [comment]
Dear @Crystal0912,

It looks like you asked the same question in a separate post, which I've now responded to: https://github.com/OpenFAST/openfast/discussions/960.

Best regards,
